server.py

```diff
@@ -45,9 +49,12 @@ def leave_room(username, room_name):
 
 # Route to get all rooms
 @app.route('/get_rooms', methods=['GET'])
-def get_rooms():
+def get_rooms(): #only returns room of the specefic user
     rooms = load_rooms()
-    return jsonify(list(rooms.keys()))
+    username = get_username()
+    user_rooms = {room_id: users for room_id, users in rooms.items() if username in users}
+    p
```

Comment:
will remove the p

Reply:
can you elaborate on what you have changed